tests/extensions/test_extensions.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import site
+
+from redash import extensions
+from tests import BaseTestCase
+
+logger = logging.getLogger(__name__)
+here = os.path.dirname(__file__)
+dummy_extension = "redash-dummy"
+
+
+class TestExtensions(BaseTestCase):
+    @classmethod
+    def setUpClass(cls):
+        dummy_path = os.path.join(here, dummy_extension)
+        site.addsitedir(dummy_path)
```

Comment:
Probably worth removing in the cleanup step?

Reply:
Yep, done!